global_planner/src/planner_core.cpp

```diff
@@ -437,4 +438,25 @@ void GlobalPlanner::publishPotential(float* potential)
     potential_pub_.publish(grid);
 }
 
+void GlobalPlanner::publishInscribedRadius() const
+{
+    double inscribed_radius, circumscribed_radius;
+    costmap_2d::calculateMinAndMaxDistances(costmap_ros_->getRobotFootprint(), inscribed_radius, circumscribed_radius);
+
+    visualization_msgs::Marker marker;
+    marker.header.frame_id = costmap_ros_->getBaseFrameID();
+    marker.header.stamp = ros::Time::now();
+    marker.ns = "inscribed_radius";
+    marker.type = visualization_msgs::Marker::CYLINDER;
```

Comment:
```suggestion
    marker.type = visualization_msgs::Marker::LINE_STRIP;
```
S: Visualize only the outline of the circle. This is cleaner and would allows to visualize both inscribed and circumscribed at the same time. You can easily do this by calling `makeFootprintFromRadius` passing the respective radii and then filling a `LINE_STRIP` with the returned points.